src/@types/did-resolver/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="../did.d.ts" />
+
+declare module 'did-resolver' {
```

Comment:
Eventually this should probably be done as part of a story to convert `did-resolver` itself to use typescript